tools/coredumper/sof-coredump-reader.py

```diff
@@ -49,6 +49,51 @@ def stdout_print(*args, **kwargs):
 	]]
 ]
 
+# Exception casues:
+# [Exception cause, excvaddr loaded]
+EXCCAUSE_CODE = [
```

Comment:
Not sure if dictionary (for potentially non-cont values) would be more appropriate than just an array indexed by the exception code.

Reply:
Sure, done